README.rst

```diff
@@ -98,12 +99,25 @@ the module ``pos_order_pricelist_change`` in the OCA "pos" repository.
         --target-version-name  12.0
         --format-patch
 
+Automatic Squashing
+----------------------
+The OCA recommand to squash automatic commit (from bot)
+This migration script will do it automatically but you need to configure
+correctly your git, so the script can apply the squash correctly
+
+You need to apply the following global config to your git
+
+.. code-block:: shell
+
+    git config --global --add rebase.instructionFormat "<%ae> %s"
```

Comment:
Can't this be added locally and automatically in the repo, if the feature is enabled?